homeassistant/components/flux_led/discovery.py

```diff
@@ -185,7 +186,7 @@ async def async_discover_devices(
     for idx, discovered in enumerate(
         await asyncio.gather(
             *[
-                scanner.async_scan(timeout=timeout, address=address)
+                create_eager_task(scanner.async_scan(timeout=timeout, address=address))
```

Comment:
This isn't ideal, but it works for now. I'll need to change the library a bit in the future to fully fix the race here